css/jquery.uls.lcd.css

```diff
@@ -102,6 +102,18 @@
 	vertical-align: middle;
 }
 
+.uls-no-results-view {
+	display: none;
+}
+
+.uls-lcd.uls-no-results > .uls-lcd-region-section {
```

Comment:
Why is this needed? Shouldn't the layout of the no-results view be fully under control of the code generating it?

Reply:
I did not like the hide class usage, I just removed it and added a marker class to the lcd as .uls-no-results - that take care of hiding and showing of regiions. We still use it for regions, but I would like to remove the usage of hide class altogether. It is just a wrapper around display:none

Reply:
Removing `hide` is good. I initially misread that this to mean we were hiding section headers in the no results template, but I see now this just hides everything else except the no results template.